system-addon/content-src/components/TopSites/TopSites.jsx

```diff
@@ -101,24 +91,21 @@ export class _TopSites extends React.PureComponent {
 
   render() {
     const {props} = this;
-    const infoOption = {
-      header: {id: "settings_pane_topsites_header"},
-      body: {id: "settings_pane_topsites_body"}
-    };
     const {editForm} = props.TopSites;
 
     return (<ComponentPerfTimer id="topsites" initialized={props.TopSites.initialized} dispatch={props.dispatch}>
-      <CollapsibleSection className="top-sites" icon="topsites" title={<FormattedMessage id="header_top_sites" />} infoOption={infoOption} prefName="collapseTopSites" Prefs={props.Prefs} dispatch={props.dispatch}>
+      <CollapsibleSection
+        className="top-sites"
+        icon="topsites"
+        title={props.intl.formatMessage({id: "header_top_sites"})}
+        extraMenuOptions={["AddTopSite"]}
+        prefName="collapseTopSites"
+        showPrefName="showTopSites"
+        eventSource={TOP_SITES_SOURCE}
+        Prefs={props.Prefs}
+        dispatch={props.dispatch}>
         <TopSiteList TopSites={props.TopSites} TopSitesRows={props.TopSitesRows} dispatch={props.dispatch} intl={props.intl} />
         <div className="edit-topsites-wrapper">
-          <div className="add-topsites-button">
-            <button
-              className="add"
-              title={this.props.intl.formatMessage({id: "edit_topsites_add_button_tooltip"})}
```

Comment:
I guess that was short lived. string already unused!

Reply:
Actually, this was only added in this 60 cycle? It should be safe to just remove now as we don't need to keep it longer for beta

Reply:
oh right. that was at the work week which was 60 already. nice

Reply:
and I guess "edit_topsites_add_button" can be removed in a few releases too